automation/src/test/scala/org/broadinstitute/dsde/test/api/ApiSpec.scala

```diff
@@ -122,23 +122,14 @@ class SamApiSpec extends FreeSpec with BillingFixtures with Matchers with ScalaF
     }
 
     "should not treat non-pet service accounts as pets" in {
+      // Use sam service account when this test moves to sam
```

Comment:
obviously this test has now moved to sam. im opening this PR so we don't forget to migrate this change over in the huge swatomation refactor. i'll consider what this comment means before merging